src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -62,8 +62,12 @@ void EquationTextBox::OnApplyTemplate()
         m_equationButton->Click += ref new RoutedEventHandler(this, &EquationTextBox::OnEquationButtonClicked);
 
         auto toolTip = ref new ToolTip();
+
         auto equationButtonMessage = m_equationButton->IsChecked->Value ? resProvider->GetResourceString(L"showEquationButtonToolTip")
                                                                         : resProvider->GetResourceString(L"hideEquationButtonToolTip");
+
+        equationButtonMessage = LocalizationStringUtil::GetLocalizedString(equationButtonMessage, EquationButtonContentIndex);
```

Comment:
>LocalizationStringUtil::GetLocalizedString(equationButtonMessage, EquationButtonContentIndex); [](start = 32, length = 94)

Maybe fold this into the previous call?  It feels a bit odd to assign a value and the immediately reassign it...
ie:
m_equationButton->IsChecked->Value ? LocalizationStringUtil::GetLocalizedString(resProvider->GetResourceString(L"showEquationButtonToolTip"), EquationButtonContentIndex) : ...

Reply:
fixed!